server/src/services/order.service.ts

```diff
@@ -0,0 +1,56 @@
+import { Op } from 'sequelize';
+import { Transaction } from 'sequelize/types';
+import { ICreateOrderRequest } from '../interfaces/orderRequest.interface';
+import Item from '../models/item.model';
+import Order from '../models/order.model';
+import OrderAddress from '../models/orderAddress.model';
+import OrderItem from '../models/orderItem.model';
+
+class OrderService {
+  public async create(transaction: Transaction, user_id: string, newOrder: ICreateOrderRequest): Promise<Order> {
+    const order = await Order.create(
+      {
+        state: 'order',
+        user_id,
+      },
+      { transaction }
+    );
+
+    const orderAddress = await OrderAddress.create(
+      {
+        order_id: order.id,
+        ...newOrder.address,
+      },
+      { transaction }
+    );
```

Comment:
```suggestion
    const order = await Order.create(
      {
        state: 'order',
        user_id,
        order_address: {
          ...newOrder.address,
        },
      },
      {
        include: OrderAddress,
        transaction,
      }
    );
```
쿼리에는 차이가 없지만,
include를 한번에 작성할 수도 있긴 하더군요 ㅎㅎ

Reply:
와.. sequelize 엄청나네요!!! create에도 include를 할 수 있는것 처음 알았습니다!

Reply:
오호 그렇군요! 감사합니다~ 코드는 수정했습니다 ㅎㅎ
다들 어디서 레퍼런스를 보시는건가요...? 저는 공식 문서 Manual을 보고 있는데 이런 부분을 찾기가 힘드네요😂

Reply:
저는 '혹시 이런거도 될까?' 싶으면 그냥 구글에 검색해봅니다!
이번에는 `sequelize create include` 로 검색했더니 바로 stackoverflow가 뜨더라구요 ㅋㅋ
https://stackoverflow.com/questions/49828855/create-with-include-sequelize